docs/book/modules/admin/pages/resources.adoc

```diff
@@ -0,0 +1,47 @@
+= Resource requirements & configuration
+
+Proper resource requests and limits configuration in Kubernetes are important to ensure the application runs
+stable and performant. Therefore, there are some reasonable resource requests in the Helm charts, which get applied
+by default.
+
+TIP: To learn mor about Kubernetes resource management, read https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/[Resource Management for Pods and Containers].
+
+If your cluster does not meet these requirements, it may be that pods cannot be started and are stuck in a `Pending`
+state. If this is the case, you can either reduce the resource requirements, at the cost of stability, or provide more
+resources to the cluster, supporting the workload.
+
+== Default resource requests
+
+The default resource request is 1 CPU and 8 GiB of RAM, for both the importer and API server deployment. There are
+no resource limits by default.
+
+== Modifying the resource requests and limits
+
+The resource requirements can be configured using the Helm Chart's "value file". After making changes, the Helm chart
+needs to be applied using the `helm update` command.
+
+The following example shows the relevant sections of the values file:
+
+[source,yaml]
+----
+modules:
+  importer:
+    resources: # <1>
+      requests:
+        memory: "12Gi"
+  server:
+    resources: # <2>
+      requests:
+        memory: "16Gi"
+        cpu: 4
+      limits:
+        cpu: 8
+----
+<1> Resource configuration for the "importer" deployment
+<2> Resource configuration for the "API server" deployment
+
+Under the `resources` field, it is possible to provide a valid Kubernetes resource configuration, which will be used
```

Comment:
This sentence is in passive voice, and should be rewritten in active voice. For example:

`The pod's container uses these resources as defined under the `resource` section for the importer feature and the API server.`